features/step_definitions/node.rb

```diff
@@ -574,3 +574,9 @@
   @result = admin.cli_exec(:rsync, source: localhost.absolutize(dst_dir), destination: "#{pod.name}:/#{dst_dir}", loglevel: 5, n: _project.name)
   raise "Error syncing files over to '#{pod.name}' pod '#{@result[:stderr]}'" unless @result[:success]
 end
+
+Given /^I select a random node$/ do
```

Comment:
@bmeng I think we can use the existing step `And I select a random node's host` instead